index.html

```diff
@@ -68,287 +68,141 @@ <h1>I'm Ekene, <br> Glad to see you!</h1>
-                                    src="./images/Snapshoot-Portfolio-(3).jpg" alt="images" class="img2">
-                            </div>
-                            <div class="sub-section3">
-                                <h2 class="mobile-ton">TONIC</h2>
-                                <h2 class="mob-face">Facebook 360</h2>
-
-                                <div class="to-do3">
-                                    <ul>
-                                        <li class="fcb">Facebook</li>
-                                        <li class="fcb"><img src="./images/Counter.jpg" alt="seperation"></li>
-                                        <li class="fcb">Full stack Dev</li>
-                                        <li class="fcb"><img src="./images/Counter.jpg" alt="seperation"></li>
-                                        <li class="fcb">2015</li>
-                                        <li class="canopy">CANOPY</li>
-                                        <li><img src="./images/Counter.jpg" alt="seperation"></li>
-                                        <li>Back End Dev</li>
-                                        <li><img src="./images/Counter.jpg" alt="seperation"></li>
-                                        <li>2015</li>
-                                    </ul>
-                                </div>
-                                <p class="to-do-para3">Exploring the future of media in Facebook's first Virtual Reality
-                                    app: a place to discover and enjoy 360 photos and videos on gear VR.
-                                </p>
-                                <div class="works">
-                                    <ul>
-                                        <li class="list"><a href="#">html</a></li>
-                                        <li class="ruby2"><a href="#">Ruby on rails</a></li>
-                                        <li class="list"><a href="#">css</a></li>
-                                        <li class="list"> <a href="#">javascript</a></li>
-                                    </ul>
-                                </div>
-                                <img src="./images/hand-cursor-1.svg" alt="cursor" class="cursor1">
-                                <div class="click24">
-                                    <button class="prj-view-btn">See Projects</button>
+            </footer>
+            <footer class="GridCardlst">
+                <section class="GridWrapperlst" id="contact">
+                    <h2>Contact me</h2>
+                    <p>
+                        If you have an application you're interested in developing,
+                        a feature that you need built, or a project that needs coding.
+                        I'd love to help with it.
+                    </p>
+                    <div class="formpart">
+                        <form action="https://formspree.io/f/xpzbdboq" method="post">
+                            <ul>
```

Comment:
- [ ] Good job so far on implementing the email validation 👍 but now when I put a valid email and submit the form, a success message just shows and the form is not submitted to form spree. Kindly make sure your form can be submitted to form spree when a valid email is used 💯 
- Your site:
![Annotation 2022-03-12 170638](https://user-images.githubusercontent.com/85113567/158025547-3f33f4fa-dce4-4da8-8568-10c4d7797014.jpg)

Reply:
HI @Kingstalux. As requested by you, I have made the changes you asked for by making sure that when my form is submitted (by putting a valid email), it gets submitted to form spree.